.scss-lint.yml

```diff
@@ -0,0 +1,256 @@
+# Default application configuration that all configurations inherit from.
```

Comment:
in der Anwendung wird kein scss-lint benötigt

Reply:
Datei wird entfernt